service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/ServiceClient.java

```diff
@@ -284,7 +337,24 @@ public void send(String deviceId, String moduleId, Message message) throws IOExc
      */
      public FeedbackReceiver getFeedbackReceiver()
      {
-        return new FeedbackReceiver(hostName, userName, sasToken, iotHubServiceClientProtocol, options.getProxyOptions(), options.getSslContext());
+         if (this.authenticationTokenProvider != null)
+         {
+             return new FeedbackReceiver(
+                     this.hostName,
```

Comment:
tabbed in 1 too many (also below)

Reply:
This is the correct tabbing in Intellij

Reply:
8 spaces is 1 tab>?

Reply:
Intellij uses 2 tabs when indenting args lists like this